<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationListWidget</class>
 <widget class="QWidget" name="StationListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stacje pomiarowe</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Lista stacji pomiarowych</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="filterComboBox"/>
     </item>
     <item>
      <widget class="QLineEdit" name="searchLineEdit">
       <property name="placeholderText">
        <string>Wprowadź kryteria wyszukiwania...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Szukaj</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="geolocateButton">
       <property name="text">
        <string>Geolokalizacja</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="stationListWidget"/>
   </item>
   <item>
    <widget class="QLabel" name="totalStationsLabel">
     <property name="text">
      <string>Łączna liczba stacji: 0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>